File: src/main/java/io/github/hdzitao/editstarters/ui/ShowErrorException.java
package io.github.hdzitao.editstarters.ui;

/**
 * 自定义的错误信息
 *
 * @version 3.2.0
 */
public class ShowErrorException extends RuntimeException {
    public ShowErrorException(String message) {
        super(message);
    }

    public ShowErrorException(String s, Throwable throwable) {
        super(s, throwable);
    }

    public static ShowErrorException internal() {
        return new ShowErrorException("!!! internal error !!!");
    }
}

File: src/main/kotlin/io/github/hdzitao/editstarters/ui/ShowErrorException.kt
package io.github.hdzitao.editstarters.ui

/**
 * 自定义的错误信息
 *
 * @version 3.2.0
 */
class ShowErrorException : RuntimeException {
    constructor(message: String) : super(message)

    constructor(s: String, throwable: Throwable) : super(s, throwable)

    companion object {
        @JvmStatic
        fun internal(): ShowErrorException {
            return ShowErrorException("!!! internal error !!!")
        }
    }
}